src/SDKs/SqlManagement/Sql.Tests/DatabaseRestoreScenarioTests.cs

```diff
@@ -301,5 +301,60 @@ public void TestDatabaseGeoRecovery()
                     });
             }
         }
+
+        [Fact]
+        public void TestDatabaseRestorePoint()
+        {
+            using (SqlManagementTestContext context = new SqlManagementTestContext(this))
+            {
+                ResourceGroup resourceGroup = context.CreateResourceGroup();
+                Server server = context.CreateServer(resourceGroup);
+                SqlManagementClient sqlClient = context.GetClient<SqlManagementClient>();
+                string restoreLabel = "restorePointLabel";
+
+                // Create database with only required parameters
+                var db = sqlClient.Databases.CreateOrUpdate(resourceGroup.Name, server.Name, SqlManagementTestUtilities.GenerateName(),
+                    new Database
+                    {
+                        Location = server.Location,
+                        Edition = "DataWarehouse",
+                        RequestedServiceObjectiveName = "DW100"
+                    });
+                Assert.NotNull(db);
+
+                CreateDatabaseRestorePointDefinition restoreDefinition = new CreateDatabaseRestorePointDefinition { RestorePointLabel = restoreLabel };
+
+                AzureOperationResponse<RestorePoint> postResponse = sqlClient.RestorePoints.CreateWithHttpMessagesAsync(
+                        resourceGroup.Name,
+                        server.Name,
+                        db.Name,
+                        restoreDefinition).Result;
+
+                Assert.True(postResponse.Response.StatusCode == System.Net.HttpStatusCode.OK ||
+                    postResponse.Response.StatusCode == System.Net.HttpStatusCode.Accepted, "Expect success in creating new restore point.");
```

Comment:
Asserts on the HttpStatusCode are not needed because the client is already doing that internally. So you can write this instead:

```
                RestorePoint postResponse = sqlClient.RestorePoints.Create(
                        resourceGroup.Name,
                        server.Name,
                        db.Name,
                        restoreDefinition);
```

similar for below